lib/ansible/modules/system/reset_ssh_session.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
Sorry, I was looking at too many PRs yesterday. You're right, it's plugins (not modules) which don't need a shebang.